### PR TITLE
[Narrative Web Report]Incorrect heading for stepmother or stepfather

### PR DESCRIPTION
**Pull request by SNoiraud**
Fixes #[10478](https://gramps-project.org/bugs/view.php?id=10478)

### CODE REVIEW

**Comment by SNoiraud** [comment]
Is it a bug or an enhancement ?

File: gramps/plugins/webreport/person.py [hunk 1]
```diff
@@ -1635,6 +1638,55 @@ def display_ind_parent_family(self, birthmother, birthfather, family,
...
+        @param: handle The handle of the father or the mother
+        self.person    The child for whom we need this result
+        """
+        bd_date = Today()
+        bd_event = get_birth_or_fallback(self.r_db, self.person)
+        if bd_event:
+            bd_date = bd_event.get_date_object()
+        for event_ref in family.get_event_ref_list():
+            event = self.r_db.get_event_from_handle(event_ref.ref)
+            if event.type == EventType.DIVORCE and \
```
Comment (Nick-Hall) [inline gramps/plugins/webreport/person.py]: PEP8 prefers implied [line continuation](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) inside parentheses.

File: gramps/plugins/webreport/person.py [hunk 2]
```diff
@@ -1635,6 +1638,55 @@ def display_ind_parent_family(self, birthmother, birthfather, family,
...
+                (event_ref.get_role() == EventRoleType.FAMILY or
```
Comment (Nick-Hall) [inline gramps/plugins/webreport/person.py]: It would be more pythonic to write:

`event_ref.get_role() in (EventRoleType.FAMILY, EventRoleType.PRIMARY)`

**Comment by Nick-Hall** [comment]
> Is it a bug or an enhancement?

I think that this is a bug.

File: gramps/plugins/webreport/person.py (hunk 1 above)
Comment (SNoiraud) [reply gramps/plugins/webreport/person.py]: OK. I suppose you prefer : 
            if (event.type == EventType.MARRIAGE and
                (event_ref.get_role() == EventRoleType.FAMILY or
                 event_ref.get_role() == EventRoleType.PRIMARY )):

File: gramps/plugins/webreport/person.py (hunk 2 above)
Comment (SNoiraud) [reply gramps/plugins/webreport/person.py]: So the solution is : 
            if (event.type == EventType.MARRIAGE and
                (event_ref.get_role() in (EventRoleType.FAMILY, EventRoleType.PRIMARY ))):

**Comment by sam-m888** [comment]
Thank you

**Pull request merged**